Description: do not clean destdir before installing libreoffice to it, as it
  already contains files from the stage packages.
Author: Olivier Tilloy <olivier.tilloy@canonical.com>

--- a/solenv/bin/ooinstall
+++ b/solenv/bin/ooinstall
@@ -77,15 +77,6 @@ if ($is_windows) {
            "-msilanguage $ENV{SRC_ROOT}/workdir/CustomTarget/instsetoo_native/install/win_ulffiles";
 }
 
-# FIXME: a better solution would be to fix installer to deal with broken symlinks
-# make distro-pack-install shufle with the SDK installation to make it LSB compliant
-# it creates symlinks from the original path to /usr/share; they are not valid with $DESTDIR
-# and potential second ooinstall call fails with strange error messages if the old tree is still there
-if ($destdir && "$ENV{DESTDIR}" ne "/" && -d "$ENV{DESTDIR}") {
-    print "Cleaning destdir...\n";
-    system ("rm -rf \"$ENV{DESTDIR}\"") && die "Failed to clean up destdir: $!";
-}
-
 print "Running LibreOffice installer\n";
 
 system ("cd $ENV{SRC_ROOT}/instsetoo_native/util ; " .
